groups/infrastructure/main.tf

```diff
@@ -21,28 +21,32 @@ terraform {
 }
 
 module "ecs-cluster" {
-  source = "git@github.com:companieshouse/terraform-library-ecs-cluster.git?ref=1.1.4"
+  source = "git@github.com:companieshouse/terraform-modules//aws/ecs/ecs-cluster?ref=1.0.225"
 
-  stack_name                 = local.stack_name
-  name_prefix                = local.name_prefix
-  environment                = var.environment
-  vpc_id                     = data.aws_vpc.vpc.id
-  subnet_ids                 = local.application_subnet_ids
-  ec2_key_pair_name          = var.ec2_key_pair_name
-  ec2_instance_type          = var.ec2_instance_type
-  ec2_image_id               = var.ec2_image_id
-  asg_max_instance_count     = var.asg_max_instance_count
-  asg_min_instance_count     = var.asg_min_instance_count
-  enable_container_insights  = var.enable_container_insights
-  asg_desired_instance_count = var.asg_desired_instance_count
+  stack_name                  = local.stack_name
+  name_prefix                 = local.name_prefix
+  environment                 = var.environment
+  aws_profile                 = var.aws_profile
+  vpc_id                      = data.aws_vpc.vpc.id
+  subnet_ids                  = local.application_subnet_ids
+  ec2_key_pair_name           = var.ec2_key_pair_name
+  ec2_instance_type           = var.ec2_instance_type
+  ec2_image_id                = var.ec2_image_id
+  asg_max_instance_count      = var.asg_max_instance_count
+  asg_min_instance_count      = var.asg_min_instance_count
+  enable_container_insights   = var.enable_container_insights
+  asg_desired_instance_count  = var.asg_desired_instance_count
+  scaledown_schedule          = var.asg_scaledown_schedule
+  scaleup_schedule            = var.asg_scaleup_schedule
+  enable_asg_autoscaling      = var.enable_asg_autoscaling
+  notify_topic_slack_endpoint = local.notify_topic_slack_endpoint
 }
 
 module "secrets" {
-  source = "./module-secrets"
+  source = "git@github.com:companieshouse/terraform-modules//aws/ecs/secrets?ref=1.0.225"
```

Comment:
Looks like the latest module version is 1.0.228

https://github.com/companieshouse/terraform-modules/tags

Reply:
Looks the "module-secrets" directory is not yet deleted. Could you remove that as we are using module reference for secrets?